templates/templates.go

```diff
@@ -17,7 +19,8 @@ import (
 	"gogs.io/gogs/internal/osutil"
 )
 
-//go:generate go-bindata -nomemcopy -nometadata -ignore="\\.DS_Store" -pkg=templates -prefix=../../../templates -debug=false -o=templates_gen.go ../../../templates/...
+//go:embed *.tmpl **/*
```

Comment:
```suggestion
//go:embed **/*
```

ditto

Reply:
just `//go:embed **/* ` will miss home.tmpl and install.tmpl file, so need add `*.tmpl` to match .tmpl file in current directory.